### PR TITLE
feat: #21 sidebar on mobile

### PR DESCRIPTION
**Pull request by htcni**
Screenshot 2022-10-02 at 18-19-03 Shuttle UI Documentation
Screenshot 2022-10-02 at 18-18-02 Shuttle UI Documentation

### CODE REVIEW

File: docs-site/docs-styles/docs-navbar.css [hunk 1]
```diff
@@ -1,67 +1,74 @@
...
-    z-index: 2;
-    
+.nav-container {
+  width: 100%;
+  height: 8.5rem;
+  background-color: var(--PRIMARY-COLOR);
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 5vw;
+  border-radius: 0 0 0px 0;
```
Comment (Maianki) [inline docs-site/docs-styles/docs-navbar.css:8]: query: Is border-radius: 0 required here?

File: docs.html [hunk 2]
```diff
@@ -32,11 +32,17 @@
             <li>
                 <a class="nav-link md-ht-1" id="dark-mode-toggle" role="button" aria-label="toggle dark mode"><i class="fas fa-moon"></i></a>
             </li>
+            <button class="btn-hamburger btn btn-primary" data-js="toggle-sidebar"><i class="fas fa-bars"></i></button>
+              <!-- <div class="btn-hamburger md-ht-1" data-js="toggle-sidebar">
+                <i class="fas fa-bars"></i>
+              </div> -->
```
Comment (Maianki) [inline docs.html]: Can remove this commented code from here.

**Review by Maianki — CHANGES_REQUESTED** [review]
Great work on the implementation @htcni . Added few minor changes please go through them.

File: docs-site/docs-styles/docs-navbar.css (hunk 1 above)
Comment (htcni) [reply docs-site/docs-styles/docs-navbar.css:8]: I haven't made any changes in navigation. The changes above are indentation difference.

Comment (Maianki) [reply docs-site/docs-styles/docs-navbar.css:8]: cool got it.

File: docs.html (hunk 2 above)
Comment (htcni) [reply docs.html]: Hey, @Maianki I have removed the comments.

**Comment by htcni** [comment]
Hey, @Maianki  I also fixed the background color on the sidebar.

File: docs.html (hunk 2 above)
Comment (Maianki) [reply docs.html]: Thanks

**Comment by Maianki** [comment]
> Hey, @Maianki I also fixed the background color on the sidebar.

Thanks for letting me know this.

**Comment by Maianki** [comment]
@htcni Thanks for contributing. I have merged your PR.

**Pull request merged**